vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,399 @@
+
+The `map_peptides_on_structure()` function is capable of mapping a continuous value to a structure. In this case that values are again scaled between 50 and 100. In this case it is possible that values fall in between 50 and 100 which are displayed as a colour gradient step in between the colour in the middle of the gradient (50) and the colour at the end of the gradient (100). 
+
+_Note: If multiple structures are mapped at the same time, by default each structure scales its map values between 50 and 100 based on its minimum and maximum score. It is however possible to use one scale for all structures that are mapped with the `scale_per_structure` argument of the `map_peptides_on_structure()` function. In this case only one of the structures (the one containing the highest score) will have the 100 value and all other structures a value lower than that as their maximum value depending on their highest score. This means that the scale of colours between both structures is comparable._
+
+```{r calculate_and_map_scores, eval = test_protti, echo = FALSE, fig.align = "center", out.width = "100%"}
+# Calculate the amino acid score
+amino_acid_score <- calculate_aa_scores(
+  data = ptsi_pgk_peptide_structure_positions,
+  protein = pg_protein_accessions,
+  diff = diff,
+  adj_pval = adj_pval,
+  start_position = start,
+  end_position = end,
+  retain_columns = c(pdb_ids, auth_asym_id)
+)
+
+# Map the score on structure
+map_peptides_on_structure(
+  peptide_data = amino_acid_score,
+   uniprot_id = pg_protein_accessions,
+   pdb_id = pdb_ids,
+   chain = auth_asym_id,
+   start_in_pdb = residue,
+   end_in_pdb = residue,
+   map_value = amino_acid_score,
+   file_format = ".cif",
+   export_location = "~/Desktop"
+ )
+```
+
+While the previous method of displaying only significantly changing peptides that have a minimum fold change is useful for the identification of regions that change the most, it is not robust to false positive peptides. The score on the other hand will only be high if an amino acid is consistently part of significant and differentially abundant peptides. The problem of the score is that it is hard and visually even impossible to interpret what a certain score means. This is due to the fact that the same score can be caused by two completely different scenarios. On the one hand an amino acid might be associated with peptides that have a low but highly significant change. On the other hand it can be associated with peptides that have a strong but less significant change. Both scenarios could yield the same amino acid score. Therefore, the score should not be misinterpreted as describing the extend of the change but should rather be seen as a probability that a certain amino acid is really structurally affected.
```

Comment:
"extent" instead of "extend"

Reply:
I would be suuuuper careful with what you write here. Maybe this should be checked at least with Patrick, if not even with Paola